variable/test/reduce_various_test.cpp

```diff
@@ -47,6 +47,22 @@ TEST(ReduceTest, min_max_with_variances) {
                                  Variances{5, 7}));
 }
 
+TEST(ReduceTest, min_max_empty_dim) {
+  const auto var = makeVariable<double>(Dims{Dim::X, Dim::Y}, Shape{2, 0},
+                                        Values{}, Variances{});
+  EXPECT_EQ(max(var, Dim::X), makeVariable<double>(Dims{Dim::Y}, Shape{0},
+                                                   Values{}, Variances{}));
+  const auto highest = std::numeric_limits<double>::max();
+  EXPECT_EQ(max(var, Dim::Y),
+            makeVariable<double>(Dims{Dim::X}, Shape{2},
+                                 Values{-highest, -highest}, Variances{0, 0}));
```

Comment:
Why did you use `-highest` instead of `lowest`?

Reply:
To ensure I did not mistunderstand `min` vs. `lowest`.

Reply:
Hm, doesn't feel like this should be part of this test

Reply:
https://en.cppreference.com/w/cpp/types/numeric_limits/lowest says it is guarenteed for std floating point types.